### ISSUE TITLE
No comment on pull request for code coverage?

### ISSUE THREAD

**Issue by oerp-odoo** [barecheck/code-coverage-action#288]
Not sure if I am doing this correctly, but how do I make it to comment on PR to show coverage report? Is it only possible on `pull_request` event?

I see in action logs that it does run successfully:

```
lcov-file: src/common/test/coverage/coverage.lcov
Current code coverage: 96.74%
Base branch code coverage: 0%
Code coverage diff: 96.74%
Code coverage diff: 96.74%
minimum-ratio: NaN
minimum-ratio is disabled for this workflow
```

But no PR comment is made.

My action looks like this:

```yaml
    - name: Generate Code Coverage report as comment
      if: inputs.with_coverage == 'true'
      # id: code-coverage
      uses: barecheck/code-coverage-action@v1
      with:
        barecheck-github-app-token: ${{ inputs.coverage_token }}
        lcov-file: src/common/test/coverage/coverage.lcov
        # base-lcov-file: "./coverage/base-lcov.info"
        send-summary-comment: true
        show-annotations: "warning"
        app-name: ${{ inputs.project }}
```

P.S. all of my related workflows are triggered by `push` event, not pull_request.

**Comment by vitaliimelnychuk** [comment]
Hi @oerp-odoo, I just wanted to let you know that the action now supports 'push' events and you can pass Pull request ID from your pipeline where you want comment to be. Here is an example:
https://github.com/barecheck/code-coverage-action?tab=readme-ov-file#push-events-support
